src/uct/ib/rc/base/rc_iface.h

```diff
@@ -86,11 +86,14 @@
     _desc->super.buffer    = _result; \
     _desc->super.user_comp = _comp;
 
-#define UCT_RC_IFACE_IS_FLUSH_REMOTE(_iface, _name) \
-    UCT_CHECK_PARAM(uct_rc_iface_is_flush_remote(_iface), \
-                    "%s %p on %s: flush(remote) is not supported", \
-                    UCS_PP_QUOTE(_name), \
-                    _iface, uct_ib_device_name(uct_ib_iface_device(&(_iface)->super)))
+#define UCT_RC_IFACE_IS_FLUSH_REMOTE(_condition, _ep, _iface, _name) \
```

Comment:
1. rename to UCT_RC_IFACE_CHECK_FLUSH_REMOTE
2. use 2 space lines in H file

Reply:
> * rename to UCT_RC_IFACE_CHECK_FLUSH_REMOTE

renamed

> * use 2 space lines in H file

but it out of code style from rest of file. added space line